package.json

```diff
@@ -1,6 +1,15 @@
 {
   "name": "explore-education-statistics",
   "private": true,
+  "packageManager": "pnpm@8.5.1",
+  "dependencies": {
+    "husky": "^7.0.4",
+    "lint-staged": "^10.4.0"
+  },
```

Comment:
The main reason to keep these in `dependencies` is due to the fact that there is a `prepare` script in the package.json. So when we try to run `pnpm --filter=explore-education-statistics-frontend... --prod install`, the command will fail due to husky not being installed

Reply:
Think we might be able to fix this by appending the `--ignore-scripts` flag to the install command. If that works, we should be able to move them back to `devDependencies`.

If not, would suggest we at least move the `dependencies` block next to the `devDependencies` block, instead of having them separated by `engines`.